src/table/components/TableWrapper.tsx

```diff
@@ -38,15 +38,15 @@ export default function TableWrapper(props) {
   const focusedCellCoord = useContextSelector(TableContext, (value) => value.focusedCellCoord);
   const setFocusedCellCoord = useContextSelector(TableContext, (value) => value.setFocusedCellCoord);
   const shouldRefocus = useRef(false);
-  const tableContainerRef = useRef();
-  const tableWrapperRef = useRef();
+  const tableContainerRef = useRef<HTMLDivElement>();
+  const tableWrapperRef = useRef<HTMLDivElement>();
```

Comment:
```suggestion
  const tableContainerRef = useRef<HTMLDivElement | undefined>();
  const tableWrapperRef = useRef<HTMLDivElement | undefined>();
```

Reply:
it is not needed.
![Screenshot 2022-08-18 at 14 36 44](https://user-images.githubusercontent.com/4471489/185396275-6c21ce40-17b9-4287-a5b1-11501ff1b193.png)

Reply:
ah ok, it takes care of that, nice